source/common/stats/stats_impl.h

```diff
@@ -103,6 +115,8 @@ struct RawStatData {
 
 private:
   static const size_t DEFAULT_MAX_NAME_SIZE = 127;
```

Comment:
Should this be DEFAULT_MAX_OBJ_NAME_SIZE? Wondering if it would be better to have all max name size parameters be derived.